tests/Carbon/IsTest.php

```diff
@@ -977,7 +977,7 @@ public function testHasFormatWithSingleLetter($letter)
     {
         $output = Carbon::now()->format($letter);
 
-        if ($output === '1000' && $letter === 'v' && version_compare(PHP_VERSION, '7.2.12', '<')) {
+        if ($output === '1000' && $letter === 'v' && PHP_VERSION_ID < 70212) {
```

Comment:
I might be silly, but I struggle to read it (confuse with 7.0; on 3.x maybe we can use the PHP 7.4 syntax for clarity and write `7_02_12`) and I would prefer to keep the version as readable as possible. Mostly in tests where we don't care to much about being fast to execute it.